R/cre.R

```diff
@@ -40,7 +41,8 @@ cre <- function(y, z, X, ratio_dis, ite_method_dis, ite_method_inf,
     }
   }
   if (class(ratio_dis) != "numeric" | !dplyr::between(ratio_dis, 0, 1)) stop("Invalid 'ratio_dis' input. Please input a number between 0 and 1.")
-  if (class(ntrees) != "numeric") stop("Invalid 'ntrees' input. Please input a number.")
+  if (class(ntrees_rf) != "numeric") stop("Invalid 'ntrees_rf' input. Please input a number.")
+  if (class(ntrees_gbm) != "numeric") stop("Invalid 'ntrees_gbm' input. Please input a number.")
```

Comment:
Again, this is hardcoding. We need to check the argument based on the approach that is provided by the user.